unstructured_inference/inference/elements.py

```diff
@@ -208,18 +205,15 @@ def extract_text(
         objects: Optional[Collection[TextRegion]],
         image: Optional[Image.Image] = None,
```

Comment:
The `image` parameter can be removed.

Reply:
i can remove how image is used in extract_text but seems like i still have to pass it to the function (i can pass a None to it):
in
```
@dataclass
class LayoutElement(TextRegion):
    type: Optional[str] = None
    prob: Optional[float] = None
    image_path: Optional[str] = None

    def extract_text(
        self,
        objects: Optional[Collection[TextRegion]],
        image: Optional[Image.Image] = None,
        extract_tables: bool = False,
    ):
        """Extracts text contained in region"""
        text = super().extract_text(
            objects=objects,
            image=image,
            extract_tables=extract_tables,
        )
        if extract_tables and self.type == "Table":
            self.text_as_html = interpret_table_block(self, image)
        return text
   ```
   the child class  `LayoutElement` is using image in `extract_text` and signature of `extract_text` in child class `LayoutElement` and parent class `TextRegion` should be the same, otherwise i see `make check` error: `Signature of "extract_text" incompatible with supertype "TextRegion"  [override]`